packages/coqide/coqide.8.4pl2/opam

```diff
@@ -34,7 +34,7 @@ synopsis: "IDE of the coq formal proof management system"
 extra-files: [
   [
     "MAKEFILE_remove_useless_for_coqide.patch"
-    "md5=4726acb2340f2d808344f8c2a7c3b685"
+    "md5=0ef819341127690e51ba1ee0f73df43e"
   ]
   ["CONFIGURE_allow_make4.patch" "md5=de289bd72e77531351acf47e3575ddc9"]
```

Comment:
Should this (and the next one) also have an updated md5?

Reply:
Indeed, my sed commands were not powerful enough. According to Travis, some versions of coq (not coqide) also have the same problems. Will have a look.

Reply:
I have just pushed a version with correct md5 for patch files. I have also added fixes for some versions of coq.